gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=2.3.0
```

Comment:
for context: as discussed between me and @Lindronics , even if we are making the `CreatePaymentResponse` class `abstract` (thus preventing its instantiation), we don't consider this a breaking change